### DISCUSSION TITLE
User level secrets like organization?

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#27115] [Actions]
<p>Organization level secrets can be defined. However, It looks not arrival for users. Is there way to define user level secrets like organization or is there any plan to add it?</p>
<p>I want to define user-level secrets for deployment tokens.</p>
<p>search keywords:</p>
<ul>
<li>user secrets like organization</li>
<li>user level secrets</li>
</ul>

**Comment by mickeygousset** [comment, marked as answer]
<p>You can currently only define secrets at three levels:</p>
<p>Organization</p>
<ul>
<li>Repository
<ul>
<li>Environment</li>
</ul>
</li>
</ul>
